core/src/i18n/en.yaml

```
@@ -105,20 +121,22 @@ namespaces:
       title: Healthy Resources
   title: Namespaces
   tooltips:
-    container-memory-limit: Define memory constraints for individual containers in your namespace.
-    create: Select this option to disable istio to mediate all communication between the pods in your namespace.
+    container-memory-limit: Define memory constraints for individual containers in your Namespace.
+    create: Select this option to disable Istio to mediate all communication between the Pods in your Namespace.
     memory-examples: Use plain value in bytes (128974848) or suffix equivalents (like 129e6, 129M, 123Mi).
-    total-memory-limit: Define constraints that limit total memory consumption in your namespace.
+    total-memory-limit: Define constraints that limit total memory consumption in your Namespace.
 oauth2-clients:
-  title: OAuth2 Clients
+  title: OAuth2Clients
```

Is this intentional? Are you referring to [this library](https://github.com/nxtbgthng/OAuth2Client)? 
'Cause if we just mean clients in the general sense, it should be two words. 
See also [our Kyma documentation](https://kyma-project.io/docs/components/api-gateway/#tutorials-expose-and-secure-a-service-register-an-o-auth2-client-and-get-tokens).
```suggestion
  title: OAuth2 clients
```